### DISCUSSION TITLE
一般功能投票几天后才会做

### DISCUSSION THREAD

**Discussion by Chormore-EDGE** [Meloong-Git/PCL#1600] [提问] [answered]
想问问什么时候确定要做的投票

**Comment by WForst-Breeze** [comment, marked as answer]
可能会结合以下因素考虑此功能是否提上日程：
- **票数**较高
- 根据现有情况决定的此功能的**必要性程度**
- 该功能的实现**难度**（难度较高的、制作周期较长的可能会搁置）

大多数还是由龙猫个人主观判断的，再结合以上几条综合考虑可行性，然后会开始制作此内容

**Comment by LTCatt** [comment]
功能投票会选择票数较高（不是最高）的制作，且通常会一次性制作数个相关联的。
一般都需要等待一个月以上乃至一年，票数太低的话一直都不会做 :(
